lib/data/doveadm.data.js

```diff
@@ -124,6 +124,8 @@ async function normalizeDoveadm(doveadm) {
 			for (const [k2, v2] of Object.entries(v.args)) {
 				if (!v2.hidden) {
 					args.push({
+						/* Undefined examples will resolve to undefined. */
```

Comment:
What's the point of this comment?

Reply:
Undefined examples don't resolve to an empty string.  That's a significant difference from some other optional config items.